cluster/deploy.sh

```diff
@@ -14,9 +14,13 @@ for i in `ls manifests/*.yaml`; do
     $KUBECTL delete -f $i --grace-period 0 2>/dev/null || :
 done
 
+$KUBECTL delete -f images/libvirtd/libvirtd-ds.yaml --grace-period 0 2>/dev/null || :
```

Comment:
The manifest `libvirtd-ds.yaml` should be moved to `manifests/` and temaplted as well - like the other pods.